src/test/java/core/basesyntax/WriterServiceTest.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.junit.Test;
+
+public class WriterServiceTest extends FruitShopTest {
+    @Test
+    public void writeReportToFile_ok() {
+        String reportText = "Some report";
+        File reportFile = writerService.writeReportToFile(reportText);
```

Comment:
You can create these objects in class (not in method) and initialize these in method @BeforeClass. And do this for all your test classes